### ISSUE TITLE
Find a logo and a branding

### ISSUE THREAD

**Issue by thomaskuntzz** [thomaskuntzz/wikicours#7]
Wikicours has no logo and no branding. This is bad because it means that the project has no identity.

The name itself, “Wikicours”, might change because I think we could find a better one (by the way, I'd love to hear your ideas [in the dedicated issue](https://github.com/thomaskuntzz/wikicours/issues/6)!). This is also a problem because how do we develop a branding without being sure of the name... 🤔😅😩

So yeah, definitely give us ideas for a better name: #6.

Anyway, so for the identity part, I think I'd better start by defining what I want the project to inspire:
 the core values of this project:
* We believe making knowledge free and accessible for all is a great way to improve society, openness is crucial
* Contributing is at everyone's reach, we want to be and look welcoming
* The content matters, it's what people are here for, we want to look serious/high-quality, invite people to work and learn efficiently
* Still have fun along the way, we want to be warm, friendly, good-natured, a little eccentric and not boring

Here are elements that I like in other websites and that I'd like to inspire myself from:
* I like [Medium's wordmark](https://fontsinuse.com/uses/18899/medium-com-2017). The font looks serious, clean, and gives a feeling of high-quality content. I think I'd like a wordmark of the same kind. All their typography is also good, inspiring the same serious aspect.
* I like the fun tone used all over [Backmarket's website](https://www.backmarket.fr), it makes them look friendly and human. I'd like to put the same touches of humour in the website

I'll be updating this as I find other things I like, please tell me what you like that we could inspire from!

**Comment by VasantJ3** [comment]
Hello @ankuhl  , I am Vasant Joseph. I am a class 12 student. I have skills in designing(Photoshop, premiere pro), presentation, and anything which is given to me, which I can do. I am sincere to my work. If there is anything which I can do to help, please inform me. This is my first time at Mozilla global sprint.
I have experience by participating in a variety of hackathons.
I am interested to help.
i can make superb LOGO

**Comment by thomaskuntzz** [comment]
Hi @VasantJ3, thanks a lot for proposing your help and congrats on joining the Global Sprint for the first time! (it's my fist time at the Sprint too 😊)

So yeah, as you can see we need a logo. I had in mind to make a **wordmark and monogram** type of logo, because it's clean and simple yet recognizable and not distracting. I like a lot [Medium's workdmark and monogram](https://www.underconsideration.com/brandnew/archives/new_logo_for_medium_done_in_house.php), I'd like to do something of the same kind. The wordmark would just read “Wikicours”, so what we need is mainly a font (and eventually a color). When trying a few online wordmark generators, I noticed I like more serif typefaces that look serious, old-style and modern at the same time. I'll post a few screenshots to show you how I'd like it to look like.

What are your thoughts on this? Do you think a wordmark is what we should go for, or do you have any other idea? Tell me what you think!

**Comment by bhargavt17** [comment]
hello,

  I am Bhargav, and iam studying in 3rd year computer science. I have skills in cloud architecture , azure, photoshop. I saw that you need a logo to be designed, iam interested in doing that work to you.
 if there is any vacancy I would surely love to work with you.

 Regards,

Bhargav
+919666366681

**Comment by VasantJ3** [comment]
i agree going is simple and effective is better than going all high and making a logo which no one understands. 
So what you are proposing is a google type logo, isn't.
Please share the screen shots which you liked.

**Comment by thomaskuntzz** [comment]
@VasantJ3 and @bhargavt17 thak you both for helping!

While the typeface doesn't have to be on Google Fonts, yes, I'm proposing a type-only wordmark kind of logo 😉

I'm collecting screenshots right now, I'll share them ASAP

**Comment by thomaskuntzz** [comment]
Ok so here we go (don't mind the colors, I didn't choose them): 

With Noe Display, the font used in Medium's logo:
<img width="514" alt="screen shot 2018-05-10 at 9 55 39 am" src="https://user-images.githubusercontent.com/16361470/39859641-c655ff0c-543a-11e8-9285-3fe2f3f48a90.png">

With logo generators:

<img width="175" alt="screen shot 2018-05-10 at 9 59 41 am" src="https://user-images.githubusercontent.com/16361470/39859645-c82f422a-543a-11e8-8a59-78c2dfebd14a.png">
<img width="150" alt="screen shot 2018-05-10 at 10 00 03 am" src="https://user-images.githubusercontent.com/16361470/39859650-cb410c3c-543a-11e8-9b17-2349cba316f9.png">
<img width="264" alt="screen shot 2018-05-10 at 10 01 40 am" src="https://user-images.githubusercontent.com/16361470/39859658-cf06c3ac-543a-11e8-98fe-e737eaab5b73.png">
<img width="241" alt="screen shot 2018-05-10 at 10 02 27 am" src="https://user-images.githubusercontent.com/16361470/39859664-d164826a-543a-11e8-85ed-032ed0cf0f45.png">
<img width="157" alt="screen shot 2018-05-10 at 10 03 32 am" src="https://user-images.githubusercontent.com/16361470/39859666-d3a83abc-543a-11e8-9651-ad8f9bd8d504.png">
<img width="340" alt="screen shot 2018-05-10 at 10 10 30 am" src="https://user-images.githubusercontent.com/16361470/39859676-da5d818c-543a-11e8-8e88-4abe8dbe07c7.png">

On this one I find the letter shapes to be a little too straight and geometric (I'd like them more curvy than this):

<img width="218" alt="screen shot 2018-05-10 at 10 08 39 am" src="https://user-images.githubusercontent.com/16361470/39859674-d860be12-543a-11e8-80ba-6ff141578822.png">

**Comment by thomaskuntzz** [comment]
These are just examples to illustrate the style I'd like. Overall, I'd like more curvy letter shapes.

**Comment by thomaskuntzz** [comment]
[The League Of Movable Type](http://theleagueofmoveabletype.com) has typefaces that are worth giving a look at I think.

**Comment by thomaskuntzz** [comment]
@VasantJ3 @bhargavt17 Do you have any thoughts? Do you think you could give a hand and help craft something like this? 🙂

**Comment by bhargavt17** [comment]
Hello ,
    I would send you some  3d effect word styles in an hour
.

about.me/tsapparapu

**Comment by VasantJ3** [comment]
I too will give some logo's.

**Comment by thomaskuntzz** [comment]
Great, excited to see what both of you guys come up with! Tanks for your help!
-------- Message d'origine --------

**Comment by bhargavt17** [comment]
about.me/tsapparapu

**Comment by thomaskuntzz** [comment]
Errr... @bhargavt17 Is that supposed to be something else than your blog? 😅

**Comment by thomaskuntzz** [comment]
@bhargavt17 Errr... Sorry, I don't get it, what is this supposed to be?

**Comment by VasantJ3** [comment]
Sorry @thomaskuntzz,  I have been working for the past 18 hours on your logo and other stuff for other projects. I almost finished. It's 4:30 in the morning here.  I am just going for a 1 hour sleep. After I wake up, I will just finalize it and sent it over. Where do you want me to sent it.

**Comment by VasantJ3** [comment]
@thomaskuntzz , Please tell where i am supposed to sent the logo. please share a email iD

**Comment by thomaskuntzz** [comment]
Hi! Sorry for the delay, just waking up here 😉😉

@VasantJ3 Do you mind if I upload here what you emailed me, so we can discuss publicly?

**Comment by VasantJ3** [comment]
![4](https://user-images.githubusercontent.com/39129698/39913544-e18e5bee-551f-11e8-94a5-84c41c3fd257.png)
![3](https://user-images.githubusercontent.com/39129698/39913545-e1cb7434-551f-11e8-84b3-867db8103274.png)
![1](https://user-images.githubusercontent.com/39129698/39913547-e22a8546-551f-11e8-9528-d02c1cde2748.png)
![2](https://user-images.githubusercontent.com/39129698/39913548-e2648b4c-551f-11e8-9b9c-43484a300bbf.png)
I  personally like 1.

**Comment by VasantJ3** [comment]
Sorry I like the third one

**Comment by thomaskuntzz** [comment]
Okay, so first off thanks a lot for your help!

I prefer the first and second one, I find them more sleek (less thick than the two others) and in line with the feeling I'd like them to inspire. I like the font and the idea of the graduation cap, however I don't like the color. I would prefer some shade of violet, or blue, or just black and dark grey. At least something that inspires work, focus, quality and seriousness.

I like the differentiation between the word "wiki" and "cours". I'm wondering if this differentiation should be only with the color, only with the font style (italic/normal), or with both. I think I would prefer it in all italic or all normal and just the word "cours" in a slightly lighter shade of violet (or blue or black or something else, depending on the color we choose).

What do you think (and by the way what font do you use)? 🙂

**Comment by VasantJ3** [comment]
Ya i will get that done.
Fonts i don't remember, after checking i will share them with you.

**Comment by thomaskuntzz** [comment]
Great! What are your thoughts about the color and the differentiation between the two words? 🙂

**Comment by VasantJ3** [comment]
I like the differentiation BTW the words. 
Colours-what is your favorite colors.
Also are you really 15?!

**Comment by thomaskuntzz** [comment]
For the differentiation between the words, I was asking if you prefer when it's done using only color (with the whole "Wikicours" word in the same font style, be it regular, bold or italic), when it's done using only font style (by using a combination of italic/bold/regular font on the words) or when it's done using both.

For the color, I was also asking you what your thoughts were. I personally would like something that looks serious and calm, something that inspires work, focus and quality. I thought we could use either violet, blue or black for such a color, do you think there is another color (or combination of colors) that could meet those criteria?

And I just turned 16 two days ago but yes 😊
